src/components/organisms/Room/LocalParticipant.tsx

```diff
@@ -1,9 +1,6 @@
-import React, { useEffect, useState } from "react";
```

Comment:
in this file: let's use named export instead of default one.

Reply:
You mean to change `React.FC` to `FC` and first line to
```typescript
import React, { FC } from "react";
```
right?

Reply:
not really. I couldn't comment on the lines I meant (because they were not changed in your PR). 

my request was to replace 
```ts
const LocalParticipant: React.FC<ParticipantProps> = ({}) => {};

export default LocalParticipant;
```

to 
```ts
export const LocalParticipant: React.FC<ParticipantProps> = ({}) => {};
```

in the project we usually use named exports (not default).

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2221/commits/38cc1cb1569924f337d9cc44a131f843e8658c1f